packages/cypress-angular-dev-server/src/lib/find-target-options.ts

```diff
@@ -53,9 +53,12 @@ function _findOptions({
   /* Standalone project configuration case: */
   if (typeof workspaceDef?.projects?.[project] === 'string') {
     const standaloneDef = JSON.parse(
-      readFileSync(resolve(root, workspaceDef.projects[project]), {
-        encoding: 'utf-8',
-      })
+      readFileSync(
+        resolve(root, `${workspaceDef.projects[project]}/project.json`),
```

Comment:
This is the only change to the code. The other updates are to add a test and to fix the tests that were failing. I am using windows which gives a different path "\\" instead of "/" and this was causing some of the tests to fail,

Reply:
Nice, thanks for the fix, appreciated!